CHANGES_NEXT_RELEASE

```diff
@@ -10,3 +10,4 @@
 - Fix: enforcing "http" or "https" schema and better detection of missing port at URLs parsing (Issue #1652)
 - Add: POST /v2/subscriptions operation (without expression evaluation yet) (Issue #1316)
 - Add: DELETE /v2/subscriptions operation (Issue #1654)
+- Fix: fixed a bug about empty strings in compounds (Issue #1642)
```

Comment:
I'd suggest the following (more precise):

"Fix: fixed wrong interpretation of empty string keyvalues as empty objects in compounds (Issue #1642)".